data/geany.glade

```diff
@@ -1563,6 +1564,69 @@
                                               </packing>
                                             </child>
                                             <child>
+                                              <object class="GtkAlignment" id="alignment15">
+                                                <property name="visible">True</property>
+                                                <property name="can_focus">False</property>
+                                                <property name="left_padding">12</property>
+                                                <child>
+                                                  <object class="GtkHBox" id="box_show_symbol_list_children">
+                                                    <property name="visible">True</property>
+                                                    <property name="can_focus">False</property>
+                                                    <property name="tooltip_text" translatable="yes">Default symbols sorting mode</property>
+                                                    <property name="spacing">12</property>
+                                                    <child>
+                                                      <object class="GtkLabel" id="label24">
+                                                        <property name="visible">True</property>
+                                                        <property name="can_focus">False</property>
+                                                        <property name="label" translatable="yes">Default sort mode:</property>
```

Comment:
Hum, just thinking, maybe @elextr would say it should rather be "Default sorting mode" here too?  I'm not great with English subtleties, so what do I know :)